### PR TITLE
#37 Applying for volunteering for an offer

### PR DESCRIPTION
**Pull request by filipgorczynski**

### CODE REVIEW

File: views.py [hunk 1]
```diff
@@ -19,9 +19,10 @@
 
 from . import models
 from volontulo.forms import CreateOfferForm
+from volontulo.forms import OfferApplyForm
 from volontulo.forms import ProfileForm
 from volontulo.forms import UserForm
-from volontulo.models import UserProfile
+from volontulo.models import UserProfile, Offer
```
Comment (magul) [inline views.py]: one import, one line

File: templates/volontulo/offer_apply.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+{% block title %}Dodaj ofertę wolontariatu{% endblock %}
+
+{% block content %}
+    <h2>Zgłoś chęć uczestnictwa w wolontariacie</h2>
+
+    <form action="{% url 'offer_apply' offer_id %}" method="post" role="form">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <button class="btn btn-primary">Zgłoś chęć współpracy</button>
```
Comment (Yog) [inline templates/volontulo/offer_apply.html]: isn't `type="submit" name="submit"` missing here ?

File: views.py (hunk 1 above)
Comment (Yog) [inline views.py]: one file to rule them all

**Comment by magul** [comment]
"This branch has conflicts that must be resolved" ??

**Pull request merged**